src/gggenome.jl

```diff
@@ -0,0 +1,142 @@
+# GGGenome
+# ======
+#
+# APIs for GGGenome.
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+"""
+GGGenome module.
+The `GGGenome` module provides a programming interface to GGGenome, 
+a ultrafast DNA sequence search service, hosted by DBCLS.
+Two functions are exported from this module:
+1. `gggenome`: DNA sequence search
+2. `gggenome_dblist`: List of available databases
+See "GGGenome Help" (https://gggenome.dbcls.jp/en/help.html) for more details.
+"""
+module GGGenome
+
+export
+    gggenome,
+    gggenome_dblist
+
+
+import HTTP
+
+const baseURL = "https://gggenome.dbcls.jp/"
+const dblistURL = "https://raw.githubusercontent.com/meso-cacase/GGGenome/master/DBlist.pm"
+
+
+# APIs of GGGenome
+# -------------------
+
+"""
+    gggenome(; params...)
+Retrieve results of gggenome search of a query sequence.
+Required parameters: 
+	query		String. Nucleotide sequence, case insensitive.
+Optional parameters:
+	db			String. Target database name. hg19 if not specified.
+				Full list of databases: https://gggenome.dbcls.jp/en/help.html#db_list
+	k			Integer. Maximum number of mismatches/gaps. 0 if not specified.
+	strand		 '+' ('plus') or '-' ('minus') to search specified strand only.
+	format		[html|txt|csv|bed|gff|json]. html if not specified.
+	timeout		Integer.
+	output		"toString": String.
+				"extractTopHit": Extract top hit as String. (Currently, only works with format="txt")
+				Otherwise: A HTTP.Messages.Response object is returned.
+	show_url	If true, print URL of REST API.
+"""
```

Comment:
This docstring will not be rendered as you expect. A docstring must be written in Markdown. Please check the guideline of documentation: https://docs.julialang.org/en/stable/manual/documentation/.